persist/sqlite/migrations.go

```diff
@@ -10,6 +10,26 @@ import (
 	"go.uber.org/zap"
 )
 
+// migrateVersion32 adds the proof height and expiration_height columns to the contracts_v2 table.
+func migrateVersion32(tx *txn, _ *zap.Logger) error {
+	_, err := tx.Exec(`
+ALTER TABLE contracts_v2 ADD COLUMN proof_height INTEGER NOT NULL;
+ALTER TABLE contracts_v2 ADD COLUMN expiration_height INTEGER NOT NULL;
+ALTER TABLE contracts_v2 DROP COLUMN window_start;
```

Comment:
Do you not want to insert the values in the new columns before dropping the old ones?

Reply:
These are v2 contracts, there shouldn’t be any yet